ci/ci/build.py

```diff
@@ -259,8 +259,18 @@ def build(self, batch, code, scope):
             context = 'context'
             init_context = 'mkdir context'
 
-        dockerfile = 'Dockerfile'
-        render_dockerfile = f'python3 jinja2_render.py {shq(json.dumps(config))} {shq(f"repo/{self.dockerfile}")} Dockerfile'
+        rendered_dockerfile = 'Dockerfile'
+        if isinstance(self.dockerfile, dict):
+            assert ['inline'] == list(self.dockerfile.keys())
+            render_dockerfile = f'echo > Dockerfile.{self.token} {shq(self.dockerfile["inline"])}'
```

Comment:
I feel like it is more readable to put the redirect after the argument.

Reply:
done.